configs/openmrs_config/roles/roles.csv

```diff
@@ -1,4 +1,12 @@
+640fb1e9-1c27-4a17-b3ad-85eda5a6fe53,Advanced User Admin,Advanced User Admin,,Add Users;View Users;Edit Users;Delete Users;Edit User Passwords,
+8c024914-db55-4b16-8de5-13494c783c0f,Basic User Admin,Basic User Admin,,Add Users;View Users;Edit Users,
+04d20272-297d-44a9-b5e7-a8d980f303ca,Advanced Patient Admin,Advanced Patient Admin,,Add patients;Edit Patients;View Patients;Delete Patients;Add Visits;Delete Visits;Add patients;Edit Patients;View Patients;Delete Patients;Add Visits;Delete Visits,
+416d4331-c217-44b1-a0f9-59d0b6c69475,Basic Patient Admin,Basic Patient Admin,,Add patients;View Patients;Add Visits,
```

Comment:
The role name is somewhat counter intuitive... What is it about appending _Admin_ to the name? Is this the specified requirement? I'd make the following changes to make it more meaningful...

`Advanced User Admin` ➜ `Advanced User Role`
`Basic User Admin` ➜ `Basic User Role`
`Advanced Patient Admin` ➜ `Advanced Provider Role`
`Basic Patient Admin` ➜ `Basic Provider Role`